packages/jaeger-ui/src/components/common/ExternalLinks.tsx

```diff
@@ -39,25 +39,22 @@ const LinkValue = (props: {
 );
 
 // export for testing
-export const linkValueList = (links: Link[]) => (
-  <Menu>
-    {links.map(({ text, url }, index) => (
-      // `index` is necessary in the key because url can repeat
-      // eslint-disable-next-line react/no-array-index-key
-      <Menu.Item key={`${url}-${index}`}>
-        <LinkValue href={url}>{text}</LinkValue>
-      </Menu.Item>
-    ))}
-  </Menu>
-);
+export const linkValueList = (links: Link[]) => {
+  const menuItems: MenuProps['items'] = links.map(({ text, url }, index) => ({
+    label: <LinkValue href={url}>{text}</LinkValue>,
+    key: `${url}-${index}`,
```

Comment:
I am curious - is there any global uniqueness requirement for these keys in the overall page? If not, why not just use something simpler for a key, like `item-${index}`?

Reply:
I'm not sure about the global uniqueness, but afaik, the scope of uniqueness is limited to the current page that we are at. `${url}-${index}` was used previously too, maybe due to any purpose.